Source/WebKit/Shared/WebsiteData/WebsiteDataType.h

```diff
@@ -50,6 +50,7 @@ enum class WebsiteDataType : uint32_t {
 #endif
     FileSystem = 1 << 19,
     BackgroundFetchStorage = 1 << 20,
+    ScreenTime = 1 << 21,
```

Comment:
This should probably be guarded by `ENABLE(SCREEN_TIME)`, as should quite a few places in this patch.